### PR TITLE
docs: add VSCode interaction issue to bazel docs

### PR DESCRIPTION
**Pull request by vicb**

### CODE REVIEW

File: docs/BAZEL.md [hunk 1]
```diff
@@ -112,19 +112,23 @@ Apple+Shift+D on Mac) and click on the green play icon next to the configuration
 - Open chrome at: [http://localhost:9876/debug.html](http://localhost:9876/debug.html)
 - Open chrome inspector
 
-## FAQs
+## Known issues
+
+### XCode
```
Comment (alfaproject) [inline docs/BAZEL.md]: Should this be `VSCode`, or does it affect any other IDE setup?

**Review by mhevery — APPROVED** [review]

File: docs/BAZEL.md (hunk 1 above)
Comment (vicb) [reply docs/BAZEL.md]: Good catch it should probably be `VSCode & Xcode`

File: docs/BAZEL.md [hunk 2]
```diff
@@ -112,19 +112,23 @@ Apple+Shift+D on Mac) and click on the green play icon next to the configuration
...
-ERROR: /private/var/tmp/_bazel_iminar/9b8801a4939e9750a817dc0cb35bbbca/external/local_config_cc/BUILD:50:5: in apple_cc_toolchain rule @local_config_cc//:cc-compiler-darwin_x86_64: Xcode version must be specified to use an Apple CROSSTOOL
+ERROR: /private/var/tmp/[...]/external/local_config_cc/BUILD:50:5: in apple_cc_toolchain rule @local_config_cc//:cc-compiler-darwin_x86_64: Xcode version must be specified to use an Apple CROSSTOOL
 ERROR: Analysis of target '//packages/core/test/render3:render3' failed; build aborted: Analysis of target '@local_config_cc//:cc-compiler-darwin_x86_64' failed; build aborted
 ```
-To resolve the error do the following:
+
+If closing VSCode fixes the issue, you can add the following line to your VSCode configuration:
+
 ```
-bazel clean --expunge
```
Comment (alexeagle) [inline docs/BAZEL.md]: I think we need this older advice too - this morning Hans had an issue that was not fixed by the exclude but was fixed by these steps

File: docs/BAZEL.md (hunk 1 above)
Comment (IgorMinar) [reply docs/BAZEL.md]: Xcode is spelled Xcode and not XCode

**Review by IgorMinar — APPROVED** [review]
otherwise lgtm, thanks for adding this!

**Comment by vicb** [comment]
@IgorMinar changed for "Xcode"
@alexeagle I added the former text back.
Thanks for the feedback.

**Pull request closed without merging**